core/src/container.cpp

```diff
@@ -345,6 +345,7 @@ Stage* ContainerBase::findChild(const std::string& name) const {
 			else if (auto* parent = dynamic_cast<const ContainerBase*>(child.get()))
 				return parent->findChild(name.substr(pos + 1));
 		}
+	RCLCPP_WARN_STREAM(rclcpp::get_logger("ContainerBase"), fmt::format("Child '{}' not found", name));
```

Comment:
I'm not convinced that the warning should be thrown here. The caller should handle a potential nullptr and may not want to have a warning issued.

Reply:
We could consider making it a debug statement? The main point of this print statement is to make debuggability better

Reply:
Yes, if that helps you.